app/assets/stylesheets/components/_card.scss

```diff
@@ -16,6 +16,7 @@
   overflow: hidden;
   background: white;
   box-shadow: 0 0 15px rgba(0,0,0,0.2);
+  margin: 14px;
```

Comment:
this will change the margin of cards everywhere (card trip also on home page for example). If you want to have a margin of cards on profile page. Then in HTML, add a class="profile-post-cards", then do Css styling override only with this card